### DISCUSSION TITLE
profile layout-sidebar text overflow

### DISCUSSION THREAD

**Discussion by ullaskunder3** [community/community#66885] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

![uiissue](https://github.com/community/community/assets/66258652/61d58ebf-f866-416c-8b10-c49d282e376f)

![Screenshot from 2023-09-12 19-36-34](https://github.com/community/community/assets/66258652/c6d3127a-ee67-4b9c-8a22-83b26e959d83)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ullaskunder3 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
